app/_scss/components/_layout.scss

```diff
@@ -60,10 +60,10 @@
 }
 
 .Layout-defaultThenLargeMarginWithHeader {
-  padding-bottom: calc(#{$Theme-spacing-default} + #{$Theme-navbar-height});
+  padding-bottom: calc(#{$Theme-spacing-default} / 2 + #{$Theme-navbar-height});
```

Comment:
Why divide this by two? If it's just so that the spacing is smaller, we can use `$Theme-spacing-small` instead.